jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -4,18 +4,28 @@
 
 ## Timeline
 
-|              | Q1 2023 | Q2 2023 | Q3 2023 | Q4 2023 | H1 2024 | June/July 2024 |
-|--------------|---------|---------|---------|---------|---------|---------|
-|              |         |         |         |         |         |         |
-| *Components* | Plan Reviews | | | | | |
-| *Platform*   | | | Plan Review  | | | |
-| *All*        | | | TCK pass w/Security Manager Disabled  | | | |
-| *All*        | | | | Milestones published | | |
-| *All*        | | | | TCK pass on Java SE 21 | | |
-| *Components* | | | | Individual Component Spec Ballots | | |
-| *Platform*   | | | | | Platform TCK pass on Java SE 21 | |
-| *Platform*   | | | | | | Platform ballot |
-| *Platform*   | | | | | | **Release** | |
+|              | Q1 2023      | Q2 2023 | Q3 2023     | Q4 2023 | Q1 2024 | Q2 2024 | June/July 2024  |
+|--------------|--------------|---------|-------------|---------|---------|---------|-----------------|
+| *Components* | Plan Reviews |         |             |         |         |         |                 |
+| *Platform*   |              |         | Plan Review |         |         |         |                 |
+| *All*        |              |         |             | TCK pass w/Security Manager Disabled |    | | |
+| *All*        |              |         |             | M1 release |      |         |                 |
+| *All*        |              |         |             |         | Wave 1, 2, 3 specs release review by 2024-01-30 | | |
```

Comment:
```suggestion
| *All*        |              |         |             |         | Wave 1, 2, 3, 4 specs release review by 2024-01-30 | | |
```

Reply:
Fixed in 790c3148b2da551ec9a02300d4883f3c50caecde.